lib/server.js

```diff
@@ -13,7 +13,7 @@ module.exports = (directory, format, width, height, framerate, horizontalFlip, v
   if (fs.existsSync(directory) === false) fs.mkdirSync(directory);
 
   // Start the camera stream
-  const raspividOptions = ['-o', '-', '-t', '0', '-w', width, '-h', height, '-fps', framerate, '-g', framerate];
+  const raspividOptions = ['-o', '-', '-t', '0', '-w', width, '-h', height, '-fps', framerate, '-g', framerate, '-n'];
```

Comment:
Can you provide a video that shows that a preview window is opened when using raspilive? That's not something I've ever seen it do before 🤔

Reply:
Sure, although I do want to know what your concern with adding this option is. It's not conflicting with any other options and is a widely discussed option.

Reply:
@rchvalbo I'm not particularly concerned with the option itself. I just want to make sure that we don't add options that aren't actually necessary.

I've only used raspi-live on a headless server (no GUI) so it's not something I've seen happen before.

Reply:
This is happening to others, resolving.